### ISSUE TITLE
Port to Chrome browser

### ISSUE THREAD

**Issue by rehman-talha** [shivaprsd/doqment#3]
Hello Shiv, I have zero coding experience and I loved your dark mode live demo. Could you please help me install this extension on chrome desktop browser? thanks!

**Comment by shivaprsd** [comment]
> Could you please add highlighter to it?

This is already tracked by #1. It is set to be added in the next release of *doqment*.

**Comment by Meghacchondo** [comment]
hey brother, I really love doqment pdf reader! and I'm still keeping firefox just because of it, but I'd really appreciate it if you could make it available for Opera & Edge.. just so you know I'm asking cuz I acknowledge your talent and hard work, I don't wanna force you nor pressure you, I'm simply requesting.. by the way, thank you for making it!

**Comment by shivaprsd** [comment]
Thank you for the appreciation brother! I also wish to make *doqment* available for all the major browsers. It would require making some small changes to the code, specific to each browser. Once the Chrome version is ready (which I hope soon), others will likely follow.. So stay tuned! 🙂

**Comment by shivaprsd** [comment]
The Chrome version is now [live](https://chrome.google.com/webstore/detail/doqment-pdf-reader/ajmcpdllaaklaocodbnllhkaflncmlog) on Chrome Webstore. 🎉

**Comment by rehman-talha** [comment]
Thank you very much for uploading. Indeed, it's an amazing extension.
